src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2486,8 +2486,9 @@ codet java_bytecode_convert_methodt::convert_instructions(
         }
         else
         {
-          assert(a_it2->second.stack.size()==stack.size());
-          stackt::const_iterator os_it=a_it2->second.stack.begin();
```

Comment:
This variable is disappearing, should it be? It probably shouldn't in this commit in any case